File: src/php/file_read.php
<?php
echo file_get_contents("../files/content.txt") . "\n";

$multiLineFile = fopen("../files/multipleLines.txt", "r");
while (!feof($multiLineFile)) {
    echo "- " . fgets($multiLineFile);
}
fclose($multiLineFile);
?>

File: src/c/file_read.c
#include <stdio.h>

int main()
{
    int c;
    FILE *file;
    file = fopen("../files/content.txt", "r");
    if (file) {
        while ((c = getc(file)) != EOF)
            putchar(c);
        fclose(file);
    }

    file = fopen("../files/multipleLines.txt", "r");
    if (file) {
        while ((c = getc(file)) != EOF)
            putchar(c);
        fclose(file);
    }
    
    return 0;
}